mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageIdManager.java

```diff
@@ -286,6 +273,7 @@ public boolean apply(MailboxId mailboxId) {
                     Mailbox currentMailbox = mailboxMapper.findMailboxById(mailboxId);
                     return belongsToCurrentUser(currentMailbox, mailboxSession);
                 } catch (MailboxException e) {
+                    mailboxSession.getLog().error("Can not retrieve mailboxPath associated with {0}", mailboxId, e);
```

Comment:
It look that you can: https://stackoverflow.com/questions/5951209/how-to-log-exception-and-message-with-placeholders-with-slf4j